packages/apps/shopify-app-remix/.changeset/new-shirts-boil.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/shopify-app-remix': minor
+'@shopify/shopify-api': minor
+---
+
+Added the new scopes api to the admin context. Included request optional scopes.
```

Comment:
I think we should have two separate changesets here, one for each package, since the changes to shopify-api are likely a patch? That would also enable us to have clearer messages.